### PR TITLE
hw01

### PR DESCRIPTION
**Pull request by YyaannaAN**

### CODE REVIEW

File: src/main/java/ua/automation/hw1/Task1.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    @AfterMethod
+    public void destroy(){
+        webDriver.quit();
+    }
+
+    @Test
+    public void task1(){
+
```
Comment (AndreyChizhikov) [inline src/main/java/ua/automation/hw1/Task1.java]: Please, delete empty lines

File: src/main/java/ua/automation/hw1/Task1.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+        webDriver.get("https://a-level.com.ua/");
+        WebElement image = webDriver.findElement(By.xpath("/html/body/header/div[2]/div/div/div/a/img[1]"));
```
Comment (AndreyChizhikov) [inline src/main/java/ua/automation/hw1/Task1.java:29]: Please, algin code

Comment (YyaannaAN) [reply src/main/java/ua/automation/hw1/Task1.java:29]: fixed

File: src/main/java/ua/automation/hw1/Task1.java (hunk 1 above)
Comment (YyaannaAN) [reply src/main/java/ua/automation/hw1/Task1.java]: fixed

**Review by AndreyChizhikov — APPROVED** [review]

**Pull request merged**